src/controller/CommissionerDiscoveryController.cpp

```diff
@@ -52,11 +53,16 @@ void CommissionerDiscoveryController::OnUserDirectedCommissioningRequest(UDCClie
     mReady = false;
     strncpy(mCurrentInstance, state.GetInstanceName(), sizeof(mCurrentInstance));
     mPendingConsent = true;
+    // TODO: print rotating ID
```

Comment:
It is already printed.

Reply:
good catch, that was a failure to save the edit in my local IDE.